package/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@cjaas/common-components",
+  "version": "1.0.0-dev.4",
```

Comment:
these files don't get generated like with uuip-publish? you need to now maintain two separate package.json files?

Reply:
Yeah, this is what Rost told me. Definitely not ideal. Let me see what I can do.

Reply:
So, Rost mentioned that because in webpack.config we have 
```
  externals: [nodeExternals({ modulesFromFile: true }), "@momentum-ui/web-components"],
```
we have more issues with just generating package.json.